.changeset/odd-bottles-fix.md

```diff
@@ -0,0 +1,10 @@
+---
+'@sl-design-system/tooltip': patch
+'@sl-design-system/shared': patch
+---
+
+Various fixes:
+- Make it possible to close a tooltip with Escape key
+- Fixed showing a tooltip not only once
```

Comment:
Please change the text to be more clear: "Fix issue where the tooltip was broken after the first show" (or something)